### DISCUSSION TITLE
Where is the Bromite browser?

### DISCUSSION THREAD

**Discussion by pozitp** [community/community#31425] [Mobile]
https://user-images.githubusercontent.com/61188367/187571651-c6c029d7-de70-48b6-9a3a-a2f0bada6f0e.mp4

Why GitHub app can't redirect to that browser?

**Comment by candyho** [comment]
Hey, do you mind elaborating what you are trying to achieve here? Are you trying to login to another service using GitHub sign in, and expect that you can complete the sign in flow in the bromite browser? Was the bromite browser your default browser?

**Comment by pozitp** [comment]
> Hey, do you mind elaborating what you are trying to achieve here? Are you trying to login to another service using GitHub sign in, and expect that you can complete the sign in flow in the bromite browser? Was the bromite browser your default browser?

Yes, I'm trying to complete sign in flow in bromite browser, and Bromite is my default browser. GitHub doesn't redirect to it and show other browsers
